mayastor/tests/common/compose.rs

```diff
@@ -566,10 +566,31 @@ impl ComposeTest {
 
         Ok(handles)
     }
+
+    pub fn down(&self) {
+        if self.clean {
+            self.containers.keys().for_each(|c| {
+                std::process::Command::new("docker")
+                    .args(&["stop", c])
+                    .output()
+                    .unwrap();
+                std::process::Command::new("docker")
+                    .args(&["rm", c])
+                    .output()
+                    .unwrap();
+            });
+
+            std::process::Command::new("docker")
+                .args(&["network", "rm", &self.name])
```

Comment:
:+1: thanks, this should solve the issue I was seeing

Reply:
It should already during Drop constructor .. but sometimes we need to control the order so I added a "manual" drop there as well... called down() +)